packages/monaco/src/typings/monaco/index.d.ts

```diff
@@ -613,89 +640,89 @@ declare module monaco.keybindings {
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/base/common/keybindingLabels.ts#L116
     export const ElectronAcceleratorLabelProvider: ModifierLabelProvider;
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/base/common/keybindingLabels.ts#L136
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/base/common/keybindingLabels.ts#L136
     export const UserSettingsLabelProvider: ModifierLabelProvider;
 
 }
 
 declare module monaco.services {
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/editor/standalone/browser/standaloneLanguages.ts#L101
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/editor/standalone/browser/standaloneLanguages.ts#L107
     export class TokenizationSupport2Adapter implements monaco.modes.ITokenizationSupport {
         constructor(standaloneThemeService: IStandaloneThemeService, languageIdentifier: LanguageIdentifier, actual: monaco.languages.TokensProvider)
         tokenize(line: string, state: monaco.languages.IState, offsetDelta: number): any;
     }
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/editor/common/services/resolverService.ts#L12
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/editor/common/services/resolverService.ts#L12
     export const ITextModelService: any;
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/platform/opener/common/opener.ts#L13
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/platform/opener/common/opener.ts#L15
     export interface OpenInternalOptions {
         readonly openToSide?: boolean;
     }
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/platform/opener/common/opener.ts#L28
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/platform/opener/common/opener.ts#L35
     export interface OpenExternalOptions {
         readonly openExternal?: boolean
     }
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/platform/opener/common/opener.ts#L38
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/platform/opener/common/opener.ts#L49
     export interface IOpener {
         open(resource: monaco.Uri | string, options?: OpenInternalOptions | OpenExternalOptions): Promise<boolean>;
     }
 
-    // https://github.com/theia-ide/vscode/blob/standalone/0.20.x/src/vs/editor/browser/services/openerService.ts#L89
+    // https://github.com/theia-ide/vscode/blob/standalone/0.23.x/src/vs/editor/browser/services/openerService.ts#L94
     export class OpenerService {
         constructor(editorService: monaco.editor.ICodeEditorService, commandService: monaco.commands.ICommandService);
         registerOpener(opener: IOpener): monaco.IDisposable;
         open(resource: monaco.Uri | string, options?: OpenInternalOptions & OpenExternalOptions): Promise<boolean>;
     }
```

Comment:
Looking at the source referenced by the comment, it seems that this interface is missing some methods (`registerValidator` and others). I am ok with this if it is intended to not fully define everything but only what we use?

Reply:
@marechal-p That is a good point - my thought here was to update references and mandatory changes, and not change interface with removed methods - as I don't know the implications of that - and also didn't add missing methods as obviously currently they aren't used - but I do not know what is preferred way of going about it - @vince-fugnitto ?